src/views/StartView.vue

```diff
@@ -10,36 +10,32 @@
 
   const serviceProviderStore: ServiceProviderStore = useServiceProviderStore();
 
+  function filterSortProviders(providers: ServiceProvider[], kategorie: ServiceProviderKategorie): ServiceProvider[] {
+    return providers
+      .filter((provider: ServiceProvider) => provider.kategorie === kategorie)
+      .sort((a: ServiceProvider, b: ServiceProvider) => a.name.localeCompare(b.name));
```

Comment:
I think as every other sorting, we should do this in the backend and receive them already sorted over here, no?

Reply:
In the refinement on monday we decided to do this in the frontend, since we already filter the serviceproviders there.
We'll probably need to change this in the future anyway, if a decision gets made to sort the providers in a specific order instead of just alphabetically

Reply:
probably the refinement before the last one where I wasn't present so I didnt get it. I'm fine with this though.